<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QWidget" name="optionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset theme="contact-new"/>
      </attribute>
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Available Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>871</width>
               <height>547</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="optionLayout">
                <property name="spacing">
                 <number>6</number>
                </property>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="optionLayout_2">
                <property name="spacing">
                 <number>6</number>
                </property>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pathTab">
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/directories.png</normaloff>:/icons/icons/directories.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>895</width>
            <height>592</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="4" column="0">
            <widget class="QCheckBox" name="moveDirectoryCheckBox">
             <property name="text">
              <string>Move infected files into DIRECTORY</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="scanReportToFileCheckBox">
             <property name="text">
              <string>Save scan report to FILE</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QLineEdit" name="moveDirectoryLineEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="selectLVDButton">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QLineEdit" name="copyDirectoryLineEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QPushButton" name="selectCFButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="selectSFFButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="scanFilesFromFileCheckBox">
             <property name="text">
              <string>Scan files from file</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="scanFilesFromFileLineEdit"/>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="copyDirectoryCheckBox">
             <property name="text">
              <string>Copy infected files into DIRECTORY</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="followFileSymlinksCheckBox">
             <property name="text">
              <string>Follow file symlinks</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="loadVirusDatabaseCheckBox">
             <property name="text">
              <string>Use the antivirus database from DIR</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="selectSCRFButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="scanReportToFileLineEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="tempFilesLineEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="followFileSymlinksComboBox">
             <item>
              <property name="text">
               <string>never</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>direct</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>always</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="selectTFButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="loadVirusDatabaseLineEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="viewButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/icons/icons/magnifierglass.png</normaloff>:/icons/icons/magnifierglass.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="followDirectorySymlinksCheckBox">
             <property name="text">
              <string>Follow directory symlinks</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="selectMDButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="followDirectorySymlinksComboBox">
             <item>
              <property name="text">
               <string>never</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>direct</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>always</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="tempFileCheckBox">
             <property name="text">
              <string>Create temporary files in DIRECTORY</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string>ClamAV Version : </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>selectSCRFButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_selectSRTFButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectSFFButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_selectSFFButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectTFButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_selectTDButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectMDButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_selectMDButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectCFButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_selectCDButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_logViewerButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>806</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectLVDButton</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>slot_selectLVDButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadVirusDatabaseCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanReportToFileCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanFilesFromFileCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tempFileCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveDirectoryCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyDirectoryCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>followDirectorySymlinksCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>followFileSymlinksCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>followDirectorySymlinksComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>722</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>followFileSymlinksComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>optionsDialog</receiver>
   <slot>writeDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>654</y>
    </hint>
    <hint type="destinationlabel">
     <x>931</x>
     <y>636</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_addButtonClicked()</slot>
  <slot>slot_removeButtonClicked()</slot>
  <slot>slot_selectLVDButtonClicked()</slot>
  <slot>slot_selectSRTFButtonClicked()</slot>
  <slot>slot_selectTDButtonClicked()</slot>
  <slot>slot_selectMDButtonClicked()</slot>
  <slot>slot_selectCDButtonClicked()</slot>
  <slot>writeDirectories()</slot>
  <slot>slot_selectSFFButtonClicked()</slot>
  <slot>slot_logViewerButtonClicked()</slot>
 </slots>
</ui>
